src/python/pants/util/contextutil.py

```diff
@@ -48,6 +48,16 @@ def get_joined_path(new_entries, env=None, env_var='PATH', delimiter=':', prepen
   return delimiter.join(path_dirs)
 
 
+def _os_encode(u, enc=sys.getfilesystemencoding()):
```

Comment:
Is using `sys.getfilesystemencoding` preferred to specifying `UTF8` explicitly?

Reply:
its definitely more correct and portable - but for our target platforms (OSX/Linux) it's usually always `utf-8`.

Reply:
could make sense to make these two helpers public and drop them in `pants.util.unicode` for general use, too.

Reply:
I think that since this is `ENV` specific, this is probably reasonable for now...